package.json

```diff
@@ -28,15 +28,18 @@
     "eslint-plugin-promise": "^6.1.1",
     "jest": "^29.7.0",
     "jest-junit": "^16.0.0",
-    "nock": "^13.5.4"
+    "nock": "^13.5.4",
+    "np": "^10.0.5"
   },
   "scripts": {
     "test": "npm run lint:check && npm run unit-tests",
     "unit-tests": "jest --ci",
     "lint:check": "eslint --ext .js .",
     "lint:fix": "eslint --ext .js --fix .",
     "api:spec:lint": "redocly lint template-registry-api.json",
-    "api:spec:generate": "redocly build-docs template-registry-api.json --output index.html"
+    "api:spec:generate": "redocly build-docs template-registry-api.json --output index.html",
+    "version": "node scripts/updateSwaggerVersion.js && npm run api:spec:generate",
+    "release": "np --no-publish"
```

Comment:
this seems a little heavy handed to build this in instead of just asking devs to do it when needed, and/or letting ghactions do it.

Reply:
I am referring to adding a dependency on np just so devs can run `npm run release` instead of `np --no-publish`

Reply:
it is a little heavy handed, although it does technically outline what the release process should be

i've seen `RELEASING.md` in some of our other repos, is that the way to go?